index.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var options = require('commander');
+var winston = require('winston');
+var retriever = require('./lib/retriever');
+var checkUsage = require('./lib/checkUsage');
+
+var logger = new winston.Logger({
+    transports: [
+      new (winston.transports.Console)()
+    ]
+  });
+
+
+//Favor source GDAL installations for ogr transformations
+process.env.PATH = '/usr/local/bin:' + process.env.PATH
+
+var esVar = process.env.ELASTICSEARCH_PORT;
+var esHost;
+var esPort;
+
+if(esVar){
+  esVar = esVar.split('//')[1].split(':');
+  esHost = esVar[0];
+  esPort = +esVar[1];
+}
+options
+  .version('0.0.1')
+  .option('-f --file <file>', 'The json data file that contains the collected data endpoints and field mappings.')
+  .option('-m --match <match>', 'A string or regular expression that the names from the <file> must contain or match')
+  .option('-h, --host <host>', 'ElasticSearch host. Defaults to localhost unless linked to a Docker container aliased to Elasticsearch', esHost || 'localhost')
```

Comment:
Some `option`s have a comma separating short and long option, but not others.  Does this have any significance?  If no, probably want to make that consistent.